hbase-server/src/main/java/org/apache/hadoop/hbase/master/DeadServer.java

```diff
@@ -140,12 +107,8 @@ public synchronized void add(ServerName sn) {
    */
   public synchronized void notifyServer(ServerName sn) {
     boolean added = processingServers.add(sn);
-    if (LOG.isDebugEnabled()) {
-      if (added) {
-        LOG.debug("Added " + sn + "; numProcessing=" + processingServers.size());
-      }
-      LOG.debug("Started processing " + sn + "; numProcessing=" + processingServers.size());
-    }
+    LOG.debug("Started processing {}; numProcessing={}; added={}",
```

Comment:
nit: Please correct me if I'm wrong. Now that we use notifyServer() as a helper in add(), calling notifyServer() doesn't mean we started processing it? Calling notifyServer() from SCP only means we started processing it, no? My point is log might be a little misleading.

Reply:
Thanks. You are right. Let me make the logging note better.

Thanks for the review.